pkg/plugin/dropreason/dropreason_linux.go

```diff
@@ -173,16 +173,24 @@ func (dr *dropReason) Init() error {
 		return err
 	}
 
-	dr.KTCPAccept, err = link.Kretprobe(intCskAcceptFn, objs.InetCskAccept, nil)
+	dr.KTCPAccept, err = link.Kprobe(intCskAcceptFn, objs.InetCskAccept, nil)
 	if err != nil {
-		dr.l.Error("opening kretprobe: %w", zap.Error(err))
-		return err
+		if errors.Is(err, os.ErrNotExist) {
+			dr.l.Warn("inet_csk_accept not found, skipping attaching kprobe to it. This may impact the drop reason metrics.")
```

Comment:
Did you confirm that this branch is getting hit on FIPS nodes?

Reply:
Yes, I could reproduce this consistently.